channel/consensus_channel/consensus_channel.go

```diff
@@ -410,3 +438,47 @@ func (c *ConsensusChannel) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// CreateFromDirectFundingObjective accepts an objective and generates a new consensus channel from it.
+// It assumes that EVERY DirectFundingObjective is for a ledger channel.
+func CreateFromDirectFundingObjective(dfo directfund.Objective) (*ConsensusChannel, error) {
```

Comment:
Originally I thought this could live in the `directFund` package but I ran into circular dependencies.

Reply:
That may be resolved by https://github.com/statechannels/go-nitro/pull/450

Reply:
I think the `directfund` package needs to know about (i.e. import `ConsensusChannel`), but `consensus_channel` should _not_ know about `directfund`.